File: 5424 - Acesso móvel a sistemas de informação/heranca_v1/heranca_v1/Aluno.swift
//
//  Aluno.swift
//  heranca_v1
//
//  Created by Gonçalo Feliciano on 26/06/2024.
//

import Foundation


class Aluno: Pessoa{
    
    var turma: String
    
    init(nome: String, idade: UInt8, turma: String) {
        
        self.turma = turma
        super.init(nome: nome, idade: idade)
    }
    
    override func getInfo() -> String {
        "\(super.getInfo())\nturma: \(self.turma)"

    }
    
}

File: 5089 - Programação – Algoritmos/heranca_java/src/Aluno.java
public class Aluno extends Pessoa{


    public Aluno(String nome, int idade) {
        super(nome, idade);
    }


    public void funcAluno() {
        System.out.println("funcAluno");
    }

    @Override
    public String toString() {
        return "Aluno";
    }
}
